AppController.m

```diff
@@ -771,9 +771,13 @@ -(BOOL)control:(NSControl *)control textView:(NSTextView *)fieldEditor doCommand
 
 -(BOOL)shouldSkip:(NSString *)contents
 {
-	NSString *type = [jcPasteboard availableTypeFromArray:[NSArray arrayWithObject:NSStringPboardType]];
+    // main type
+    NSString *type = [jcPasteboard availableTypeFromArray:[NSArray arrayWithObject:NSStringPboardType]];
```

Comment:
Of course, I had to make a suggestion that doesn't compile. But using `[pboardTypes description] ofType:@"NSStringPboardType"` would still be good I think.